python/pylibraft/pylibraft/distance/pairwise_distance.pyx

```diff
@@ -118,51 +118,63 @@ def distance(X, Y, out=None, metric="euclidean", p=2.0, handle=None):
     --------
     To compute pairwise distances on cupy arrays:
 
-    >>> import cupy as cp
-    >>> from pylibraft.common import Handle
-    >>> from pylibraft.distance import pairwise_distance
-    >>> n_samples = 5000
-    >>> n_features = 50
-    >>> in1 = cp.random.random_sample((n_samples, n_features),
+    .. code-block:: python
```

Comment:
I appreciate that this is consistent with all the other docstrings in raft right now - but I think we should probably be standardizing the other way on the numpydoc format so that we can use doctest to test these docstrings automatically (#981)

Reply:
Got it. In that case I definitely think we should use the other format. Let me create an issue for that for 23.02 as well. It's just getting too close to code freeze to do that for 22.12.

Reply:
yeah - I don't think we should change this for 22.12, but would be nice to get in for 23.02 .